components/tabstrip/dynamic-tabs.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Dynamic Tabs
+page_title: TabStrip - Dynamic Tabs
+description: Learn how to use the ActiveTabId parameter in the Telerik TabStrip for Blazor to manage dynamic tabs.
+slug: tabstrip-dynamic-tabs
+tags: telerik,blazor,tabstrip,dynamic tabs
+published: True
+position: 3
+---
+
+# Dynamic Tabs in TabStrip
+
+The Telerik TabStrip component now supports managing dynamic tabs more effectively with the introduction of the `ActiveTabId` parameter and the new [`ActiveTabIdChanged`](slug:tabstrip-events#activetabidchanged) event. These features allow users to specify or track the active tab using its unique ID, making it easier to work with dynamic tab scenarios.
```

Comment:
Try writing text that isn't time-sensitive ("now supports", "the new `ActiveTabIdChanged` event" ). Reading this paragraph in two years would feel awkward as these statements will be outdated.

```suggestion
The Telerik TabStrip component supports effective management of dynamic tabs through the `ActiveTabId` parameter and the [`ActiveTabIdChanged`](slug:tabstrip-events#activetabidchanged) event. These features allow users to specify or track the active tab using its unique ID, making it easier to work with dynamic tab scenarios.
```